framework/aster-frame/src/arch/x86/mm/mod.rs

```diff
@@ -79,22 +77,25 @@ pub unsafe fn activate_page_table(root_paddr: Paddr, flags: x86_64::registers::c
     );
 }
 
-pub static ALL_MAPPED_PTE: Mutex<BTreeMap<usize, PageTableEntry>> = Mutex::new(BTreeMap::new());
+pub(crate) static INIT_MAPPED_PTE: Once<BTreeMap<usize, PageTableEntry>> = Once::new();
```

Comment:
To be honest, this variable name confuses me a bit, because in the Linux kernel the word "PTE" is specific to the entries of the lowest level of the page table, and other words like `PMD`, `PUD`, `P4D`, and `PGD` are used for the upper levels of the page table, see the definitions [here](https://elixir.bootlin.com/linux/v6.8.1/source/arch/x86/include/asm/pgtable_64_types.h#L14).

Perhaps Asterinas follows a different convention? That's fine, but just mention that it would be better if the variable name could somehow emphasize that this `BTreeMap<usize, PageTableEntry>` stores entries of the _highest_ level of the page table.